### ISSUE TITLE
Different API version between core 1024 & binding 1280 (CS_ERR_VERSION) in 4.0.2 release

### ISSUE THREAD

**Issue by PAX523** [capstone-engine/capstone#1658]
I've downloaded the sources and pre-compiled native windows libraries. When I try to disassemble something via the Java binding from the associated sources then I obtain following error:

```
java.lang.RuntimeException: Different API version between core 1024 & binding 1280 (CS_ERR_VERSION)
```

The version settings in the binding indicate that they are incompatible (newer) with the natives.

Note: I also percepted an issue when I use the latest JDK 8 (https://bell-sw.com/pages/downloads/#/java-8-lts). It is already described here (but for later Java releases): https://github.com/aquynh/capstone/issues/1246

Please make sure, that this works as well for the latest release.

**Comment by PAX523** [comment]
The binding version is specified in:

```
capstone-4.0.2.zip/capstone-4.0.2/bindings/java/capstone/Capstone.java

  public static final int CS_API_MAJOR = 5;
  public static final int CS_API_MINOR = 0;
```

As indicated by the error message, it seems that the major version within the natives is set to 4 and minor equals 0.

**Comment by PAX523** [comment]
In commit `66808143dbd65ac518635c8cd83935df9c403954` the pointer size issue was fixed. But also the binding's major version was touched (changed from `4` to `5`). Was this your intention?

**Comment by mquigley** [comment]
@PAX523 This was a while ago, but I believe at the time, the master branch was set to version 5. 

Probably what happened was that after the last release, someone bumped the version, but then decided to move it back. But honestly I'm not sure... sounds like you just need to change the version back in the binding.

**Comment by PAX523** [comment]
Yes, that was my emergency approach. But to be honest: I'm not satisfied with such a project quality. Such things should not happen.
